distribution/pom.xml

```diff
@@ -357,5 +357,12 @@
             <artifactId>awaitility</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I think we don't need this dependency anymore after the recent changes

Reply:
i'll re-try

e53df87db89dc49bcff336f34f8f31c61ddaa59e